### ISSUE TITLE
Reader: VoiceOver's focus jumps at beginning of article list when placing the first like on a post

### ISSUE THREAD

**Issue by elettrona** [wordpress-mobile/WordPress-iOS#22065]
### Expected behavior
when pressing the "Like" button on a post, VoiceOver's focus should remain on the "like" button. 

### Actual behavior
When pressing the "Like" button on an article and it's the first like the post gets, VoiceOver's focus jumps at the beginning of the posts list, making it difficult to proceed browsing the whole Reader. Involved post must be closed, it occurs in the main Reader window.

### Steps to reproduce the behavior
1. from JetPack app, go to Reader tab. VoiceOver must be active.
2. navigate the page sequentially until you find the "like" button, on a post without any other likes. Don't open the post.
3. place the first Like to the post, so that it will get "one like". 
4. VoiceOver's cursor will immediately jump to the very first article displayed on the feed.

**Comment by elettrona** [comment]
labels: accessibility, reader, VoiceOver, bug

**Comment by twstokes** [comment]
Thank you for the report @talksina!

I was able to reproduce this and determine the regression occurred in our 23.5 release:

| 23.4 | 23.5 |
| - | - |
| <video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/2092798/f1a3d6d1-3d0a-4788-8106-b13b6b74b887" alt="Screen recording of liking a Reader post in 23.4 while using VoiceOver" /> | <video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/2092798/83a5a6ab-8e2b-474e-8b21-d0dd839df620" alt="Screen recording of liking a Reader post in 23.5 while using VoiceOver" /> |

**Comment by derekblank** [comment]
I was able to replicate the VoiceOver focus jump issue. I noted in the video examples above that there is a visual difference between 23.4 and 23.5. In 23.4, the number "1" appears to the right of next to the Star icon when something is liked. In 23.5, the label "1 Like" is added between the button toolbar and the post text, which pushes the vertical position of the toolbar down and outside of the focus area.

I also noted that when the content is not fully visible on the screen, the content jumps briefly when the Like button is pressed. In this video example, I am scrolling content to various positions where the title of the content is partially out of view, and then pressing the Like button. When this occurs, the entire piece of content is pushed down briefly, and returns to its original position. It's not very noticeable, but perhaps whatever view issue that is causing this content to jump is also the same issue that is affecting the focus of the VoiceOver. 

https://github.com/wordpress-mobile/WordPress-iOS/assets/643285/f8362c11-27c5-427f-a701-972935788048

**Comment by elettrona** [comment]
Same issue happens when you press the context menu button (found after the "like" and, from there, you press "share". 
After having shared the article wherever you want, the focus jumps to the beginning. This last one happens regardless of the number of likes/comments the article has.

**Comment by elettrona** [comment]
issue still present in 23.7.

**Comment by twstokes** [comment]
Thank you @talksina. I've assigned this issue to myself and will be addressing it soon.

**Comment by elettrona** [comment]
@twstokes updated to 23.8 today and bug still persists

**Comment by twstokes** [comment]
Thank you @talksina. I will follow up when we've merged in the fix and what version you can expect it in.

**Comment by twstokes** [comment]
@talksina a fix has been approved and should be included in version 23.9. That release can be expected around January 8th. Thank you for reporting this!
